source/crud/crud.md

```diff
@@ -815,6 +815,12 @@ database-level aggregation will allow users to receive a cursor from these colle
 
 ##### Insert, Update, Replace, Delete, and Bulk Writes
 
+###### Generated identifiers
+
+The insert and bulk insert operations described below MUST generate identifiers for all documents that do not already
```

Comment:
Noted that this is already discussed in the [client bulk write spec](https://github.com/mongodb/specifications/blob/master/source/crud/bulk-write.md#insert).

Reply:
Having this behavior discussed here for general CRUD operations makes sense to me, as the client bulk write spec follows from this broader context.

Reply:
The client bulk write spec stands on its own, except for a reference back to CRUD for [modeling unacknowledged write results](https://github.com/mongodb/specifications/blob/master/source/crud/bulk-write.md#unacknowledged-results).

With my last comment, I just meant to acknowledge that no changes were needed to the client bulk write spec since it already addressed `_id` ordering. This was in the vein of previous PRs like https://github.com/mongodb/specifications/pull/1644 that required updates to _both_ specs.